README.md

```diff
@@ -136,24 +136,24 @@ conda install pytorch-lightning -c conda-forge
 
 the actual status of 1.5 \[stable\] is following:
 
-![CI basic testing](https://github.com/PyTorchLightning/pytorch-lightning/workflows/CI%20basic%20testing/badge.svg?branch=release%2F1.5.x&event=push)
-![CI complete testing](https://github.com/PyTorchLightning/pytorch-lightning/workflows/CI%20complete%20testing/badge.svg?branch=release%2F1.5.x&event=push)
-![PyTorch & Conda](https://github.com/PyTorchLightning/pytorch-lightning/workflows/PyTorch%20&%20Conda/badge.svg?branch=release%2F1.5.x&event=push)
-![TPU tests](https://github.com/PyTorchLightning/pytorch-lightning/workflows/TPU%20tests/badge.svg?branch=release%2F1.5.x&event=push)
-![Docs check](https://github.com/PyTorchLightning/pytorch-lightning/workflows/Docs%20check/badge.svg?branch=release%2F1.5.x&event=push)
+![CI basic testing](https://github.com/Lightning-AI/lightning/workflows/CI%20basic%20testing/badge.svg?branch=release%2F1.5.x&event=push)
+![CI complete testing](https://github.com/Lightning-AI/lightning/workflows/CI%20complete%20testing/badge.svg?branch=release%2F1.5.x&event=push)
+![PyTorch & Conda](https://github.com/Lightning-AI/lightning/workflows/PyTorch%20&%20Conda/badge.svg?branch=release%2F1.5.x&event=push)
+![TPU tests](https://github.com/Lightning-AI/lightning/workflows/TPU%20tests/badge.svg?branch=release%2F1.5.x&event=push)
+![Docs check](https://github.com/Lightning-AI/lightning/workflows/Docs%20check/badge.svg?branch=release%2F1.5.x&event=push)
```

Comment:
aren't these batches old?

Reply:
can be updated in a follow up though

Reply:
Where should they point to? 1.6.x?

Reply:
ideally it should be `release/pytorch`.

@akihironitta might know

Reply:
Well we used to refer to master, and in the release is a replacement for a specific tag
https://github.com/Lightning-AI/lightning/blob/0cbfe08a41a5f517a27fd432d4770feb7bef37af/.actions/setup_tools.py#L73

Reply:
Updated them in 4bfa715!